packages/cashier/src/pages/account-transfer/account-transfer-form/account-transfer-form.tsx

```diff
@@ -57,15 +174,15 @@ const AccountOption = ({ mt5_login_list, account, idx, is_dark_mode_on }) => {
                 <Money
                     amount={account.balance}
                     currency={account.currency}
-                    has_sign={account.balance < 0}
+                    has_sign={account.balance && account.balance < 0}
```

Comment:
You can use `account?.balance < 0` instead of `account.balance && account.balance < 0`